src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -3499,7 +3499,7 @@ void Fishing_UpdateFish(Actor* thisx, PlayState* play2) {
                     }
                     D_80B7E122 = phi_v0;
                     D_80B7E0A4 = phi_v0;
-                    func_800A9F6C(0.0f, 180, phi_v0 * 3, 10);
+                    Rumble_Override(0.0f, 180, phi_v0 * 3, 10);
```

Comment:
Of course Fishing has the most complicated rumble in the game. Could a couple of these variables be named, or are they too widely used?

Reply:
I took a brief look but it was too difficult to tell at a glance what the purpose of each rumble call is, and since the functions where rumble is used have different variables for various rumble parameters they'd need to be named differently based on the purpose.